docs/linux_deployment.md

```diff
@@ -0,0 +1,515 @@
+# Deployment on Linux
+
+This guide is focused on installing Sphinx-relay on top of LND on Linux.
+
+### Preparations
+
+- Connect to your node via SSH to the user account where your LND runs (most people use `bitcoin`)
+
+```sh
+$ ssh bitcoin@ip.ad.dr.ess
```

Comment:
Is this the actual ip address? It might be worth noting to the person reading this that they might need to find this info and where they can find that

Reply:
done